.eslintrc.cjs

```diff
@@ -30,6 +30,8 @@ module.exports = {
     ],
     'prettier/prettier': 'off',
     'no-mixed-operators': 'error',
+    '@typescript-eslint/prefer-readonly': 'error',
+    '@typescript-eslint/prefer-readonly-parameter-types': ['error', { 'ignoreInferredTypes': true }],
```

Comment:
Without `ignoreInferredTypes` makes forcing all of annotations for callbacks. It looks cons wins.  😢